### DISCUSSION TITLE
Open tracing span propagation

### DISCUSSION THREAD

**Discussion by mario-imperato** [go-resty/resty#504] [Q&A] [answered]
What is the proper way to propagate a span over the http request?
Thank you

**Comment by mario-imperato** [comment]
Need to Inject into the Request.Header the span or by other means? Something along the lines below?
Thank you again.

`httpReq := client.R()

  // Transmit the span's TraceContext as HTTP headers on our
  // outbound request.
  opentracing.GlobalTracer().Inject(span.Context(), opentracing.HTTPHeaders,    
          opentracing.HTTPHeadersCarrier(httpReq.Header))
`

**Comment by FarmerChillax** [comment, marked as answer]
refer to https://github.com/go-resty/resty/discussions/648#discussioncomment-6172483
